src/api/hooks/usePolkaassemblyDiscussionDetail.ts

```diff
@@ -0,0 +1,199 @@
+import {NetworkContext} from 'context/NetworkContext';
+import gql from 'graphql-tag';
+import {print} from 'graphql/language/printer';
+import {useContext} from 'react';
+import {useQuery} from 'react-query';
+
+export function usePolkaassemblyDiscussionDetail(id: number) {
```

Comment:
There is an extra "a" in the name. 
```suggestion
export function usePolkassemblyDiscussionDetail(id: number) {
```
Also in the other places where **polkassenbly** is used